examples_extra/TextStressTest/Sample/ViewController.m

```diff
@@ -0,0 +1,214 @@
+/* This file provided by Facebook is for non-commercial testing and evaluation
+ * purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#import "ViewController.h"
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+#define NUMBER_ELEMENTS 5
+
+@interface ViewController ()
+{
+  NSMutableArray <ASTextNode *> *_textNodes;
+  NSMutableArray <UILabel *> *_textLabels;
+  UIScrollView *_scrollView;
+}
+
+@end
+
+
+@implementation ViewController
+
+- (instancetype)init
+{
+  if (!(self = [super init]))
+    return nil;
+  
+  _scrollView = [[UIScrollView alloc] init];
```

Comment:
Move all three of these into -viewDidLoad, because _scrollView being created there would be better anyway, and the arrays aren't needed until that point.  Technically we could just delete -init entirely then.